tests/test_default_qubit.py

```diff
@@ -722,20 +722,6 @@ def test_sample_values(self, qubit_device_2_wires, tol):
         assert np.allclose(s1**2, 1, atol=tol, rtol=0)
 
 
-class TestMaps:
-    """Tests the maps for operations and observables."""
-
-    def test_operation_map(self, qubit_device_3_wires):
-        """Test that default qubit device supports all PennyLane discrete gates."""
```

Comment:
Removed because the new gates are not directly supported? Why not add them in?

Reply:
With gates that support decomposition, I figured it was overkill to force default qubit to natively support all gates inside `ops/qubit.py`. Especially if we start to have same gates with different names (`U1`, `PhaseShift`).

Reply:
I thought that was one of the intentions of `default.qubit`, to provide a reference implementation of all gates?

Reply:
It was yes, back when we were keeping the core PennyLane gate set small and avoiding duplication. Now that we have `U1`/`PhaseShift`, `U3`/`Rot`, I'm not sure that makes so much sense any more.

Maybe it should be changed so that we test that `default.qubit` implements all gates with no decomposition. If this is satisfied, all user-available gates should work. I'll make a new issue for this.